lib/omnibus/packagers/msi.rb

```diff
@@ -73,9 +76,13 @@ class Packager::MSI < Packager::Base
             -cultures:en-us
             -loc "#{windows_safe_path(staging_dir, 'localization-en-us.wxl')}"
             project-files.wixobj source.wixobj
-            -out "#{windows_safe_path(Config.package_dir, package_name)}"
+            -out "#{msi_file}"
         EOH
         shellout!(light_command, returns: [0, 204])
+
+        if cert_name
```

Comment:
It would be nice for everything to just work a dev box. So that means, I should be able to tell omnibus that it shouldn't sign anything without needing to comment stuff out in the project file.